### PR TITLE
changing REST integration test command line.

### LINKED ISSUE #15124: What's the best way to run REST integration tests?
https://github.com/elastic/elasticsearch/blob/master/TESTING.asciidoc suggests 

```
gradle integTest -Dtests.filter="@Rest"
```

but that matches no tests, I've had some success with 
```
gradle integTest -Dtests.class="*.Rest*IT"
```

Is that the recommended way to run these?

### PR DESCRIPTION
**Pull request by andrejserafim**
fixed #15124.

### CODE REVIEW

File: TESTING.asciidoc [hunk 1]
```diff
@@ -290,7 +290,16 @@ The REST tests are run automatically when executing the maven test command. To r
 REST tests use the following command:
```
Comment (jasontedor) [inline TESTING.asciidoc]: The line immediately above this refers to running "maven test". Can you reword this to refer to running "gradle check"?

**Comment by jasontedor** [comment]
Left a small comment, otherwise LGTM.

**Comment by andrejserafim** [comment]
Changed.

**Comment by jasontedor** [comment]
Integrated to master in 9015d0ca73d8b7cfff6c839b643ab3367e411531. Thank you!
